### DISCUSSION TITLE
Error after changing color scheme [BUG REPORT]

### DISCUSSION THREAD

**Discussion by Dolbaeb777K** [end-4/dots-hyprland#799] [Q&A]
So, when I changed color scheme by `SUPER + COMMA` then clicking on `Fruit Salad` this errors appeared.
![изображение](https://github.com/user-attachments/assets/1463e52c-b7f2-4f11-a7d3-926f74debe2a)
This error appears for all Scheme styles.

Installation: Automated 
OS: Arch btw

**Comment by Dolbaeb777K** [comment]
hm... this errors disappeared after disable `Use gradient`

**Comment by end-4** [comment]
what if you select a wallpaper with ctrl super t? does the error go away

**Reply by Dolbaeb777K** [reply]
Maybe... After that I switched to JaKooLit's dotfiles But Then I switched to your dotfiles. But after installing I changed wallpaper in first, and this error never appeared.
